app/views/budgets/investments/edit.html.erb

```diff
@@ -0,0 +1,6 @@
+<div class="budget-investment-new row">
+  <div class="small-12 medium-9 column">
+    <h1><%= t("management.budget_investments.edit") %></h1>
+    <%= render '/budgets/investments/form_edit', form_url: budget_investments_path(@budget) %>
```

Comment:
Is there a way to reuse the `/budgets/investments/form` partial here? Then we could remove the `form_edit` file and so remove the duplication between both files :thinking:.

Reply:
hey yes, I was having problems at first with the `translatable_fields` so I decided to use a separate form, but it seems to be working now 🤷